### PR TITLE
Hb workbranch2 -EN update ISSP

### PR DESCRIPTION
**Pull request by H4516022Y**
ISSP to be updated online
PS : the date of update must be aligned with the date indicated in the last ISSP update already indicated in the ISSP text = 21th July 2022
french version update will follow and I will submit the modification

### CODE REVIEW

File: pages/account/security/information-system-security-policy/guide.en-gb.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ slug: issp
 section: Security policies
 ---
 
-**Last updated 15th February 2021**
+**Last updated 21th July 2022**
```
Comment (OVH-Team-Guides) [inline pages/account/security/information-system-security-policy/guide.en-gb.md]:
```suggestion
**Last updated 21st July 2022**
```

File: pages/account/security/information-system-security-policy/guide.en-gb.md [hunk 2]
```diff
@@ -64,25 +64,25 @@ The threats targeting OVHcloud are primarily of external origin. However, our te
...
 
 #### Employees, management of OVHcloud and shareholders
 
-OVHcloud employees design, maintain and operate systems and processes in support of OVHcloud services. Any security incident has a direct impact and generates operational disruptions. It can also challenge the reliability of customer service, the expertise and professionalism of teams. Operating secure information systems enhances OVHcloud's innovation, passion, team engagement and service quality.
+OVHcloud employees design, maintain and operate systems and processes in support of OVHcloud services. Any security incident as a direct negative impact on operations, but also challenge the value of the services, the expertise and professionalism of teams. Operating secure information systems enhances OVHcloud's innovation, passion, team engagement and service quality.
```
Comment (OVH-Team-Guides) [inline pages/account/security/information-system-security-policy/guide.en-gb.md]:
```suggestion
OVHcloud employees design, maintain and operate systems and processes in support of OVHcloud services. Any security incident has a direct negative impact on operations, but also challenge the value of the services, the expertise and professionalism of teams. Operating secure information systems enhances OVHcloud's innovation, passion, team engagement and service quality.
```

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]

File: pages/account/security/information-system-security-policy/guide.en-gb.md (hunk 1 above)
Comment (H4516022Y) [reply pages/account/security/information-system-security-policy/guide.en-gb.md]: OK for it

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @H4516022Y

**Pull request merged**